### ISSUE TITLE
Define state of graph visualisation in scheme

### ISSUE THREAD

**Issue by JeannedArk** [annefried/discanno#4]
A change in the entity model is mandatory. All example schemes have to be updated.

**Comment by JeannedArk** [comment]
- LinkSets naming ✅ 
- graph/ timeline customization ✅ 
- graph/ timeline: enabled/ disabled ✅ 
- undirected/ directed per LinkSet ❓ 
- ChainColoring ❓ 
- SpanType extend from BaseEntity ❓ 
- ColorableBaseEntity extends from BaseEntity; SpanType, Label, LinkLabel, LabelSet and LinkType extend from ColorableBaseEntity ❓ 

Coloring (all exclusive)
- chose color per targettypes (Shades mode, one shade per Label)
- chose color per label

**Comment by JeannedArk** [comment]
https://github.com/annefried/discanno/issues/48
https://github.com/annefried/discanno/issues/47

**Comment by JeannedArk** [comment]
@annefried When selecting timeline exclusively just allow "after"/ "before"/ "overlap" as LinkLabels?

**Comment by JeannedArk** [comment]
Blocked by https://github.com/annefried/discanno/issues/70

**Comment by annefried** [comment]
The link labels and directions in graph should ultimately be configurable.

**Comment by JeannedArk** [comment]
What is exactly meant with link labels should be configurable?

**Comment by annefried** [comment]
Which link label means "start node left to end node" and which link label means "start/end nodes on top of each other"

**Comment by annefried** [comment]
When changing the visualisation in the scheme editor, they don't seem to be delete but simply added. This then seems to lead to a typeError when attempting to save the scheme.

**Comment by judembo** [comment]
Changing the visualization seems to work fine for me. However, after a redeploy, loading an existing scheme from the database and then trying to save it (with or without modification) leads to a typeError. Maybe it's the same one?
I will create a new issue for this.
